molecule-runtime/src/jsTest/kotlin/app/cash/molecule/WindowAnimationFrameClockTest.kt

```diff
@@ -15,18 +15,20 @@
  */
 package app.cash.molecule
 
+import assertk.all
+import assertk.assertThat
+import assertk.assertions.isLessThan
+import assertk.assertions.isPositive
 import kotlin.test.Test
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.Job
 import kotlinx.coroutines.test.runTest
 
-@OptIn(ExperimentalCoroutinesApi::class)
 class WindowAnimationFrameClockTest {
-  @Test fun ticks() = runTest {
-    val job = Job()
-    WindowAnimationFrameClock.withFrameNanos {
-      job.complete()
+  @Test fun ticksWithTime() = runTest {
+    val frameTimeA = WindowAnimationFrameClock.withFrameNanos { it }
+    val frameTimeB = WindowAnimationFrameClock.withFrameNanos { it }
+    assertThat(frameTimeA).all {
```

Comment:
`all` is fun

Reply:
Yep. It defers throwing until the end of the block and aggregates any failures. Very useful.